release-notes/8.0/8.0.0/api-diff/Microsoft.NETCore.App/8.0.0_System.Buffers.md

```diff
@@ -0,0 +1,26 @@
+# System.Buffers
```

Comment:
System.Buffers LGTM

Primarily the new SearchValues support

CC. @dotnet/area-system-buffers